src/ApiFunction.ts

```diff
@@ -51,17 +51,17 @@ export class ApiFunction extends Construct {
    * @param filterPattern Pattern to filter by (default: containing ERROR)
    */
   private monitor(filterPattern?: IFilterPattern) {
-    const filter = new MetricFilter(this, 'MetricFilter', {
+    const errorMetricFilter = new MetricFilter(this, 'MetricFilter', {
```

Comment:
'MetricFilter' als naam is erg generiek. Kan later problemen opleveren.

Reply:
Dit is een wijziging van september die om een of andere reden weer terugkomt. De naam is hier `errorMetricFilter` (hernoemd van 'filter'), om dezelfde reden. Wordt alleen in deze functie gebruikt, functie is 15 regels of zo. Specifieker is lastig, want wordt door verschillende lambdas gebruikt, en het is een filter op errors, dus op zich wel beschrijvend?